engine/src/main/java/org/terasology/rendering/opengl/ScreenGrabber.java

```diff
@@ -102,7 +103,8 @@ public void saveScreenshot() {
             displayResolutionDependentFBOs = CoreRegistry.get(DisplayResolutionDependentFBOs.class);
         }
 
-        FBO sceneFinalFbo = displayResolutionDependentFBOs.get(FINAL_BUFFER);
+        //FBO sceneFinalFbo = displayResolutionDependentFBOs.get(FINAL_BUFFER);
+        FBO sceneFinalFbo = displayResolutionDependentFBOs.get(DEPTH_TO_RGB_FBO_URI);
```

Comment:
I assume this is only for debugging purposes?

For future, you can directly use the commend `dagNodeCommand engine:fbo.outputToScreenFbo setFbo engine:fbo.convertDepthToRGB` to display convertDepthToRGB on screen directly.

Reply:
Indeed I'd use that method rather than hijack the sceneFinalFbo. 

One day we will rather attach the output of convertDepthToRGB to the outputNode, at runtime.